assets/js/offline-search.js

```diff
@@ -0,0 +1,196 @@
+                    // Here you can specify searchable fields to the search index - e.g. individual toxonomies for you project
+                    // With "boost" you can add weighting for specific (default weighting without boost: 1)
+                    this.field('title', { boost: 5 });
+                    this.field('categories', { boost: 3 });
+                    this.field('tags', { boost: 3 });
+                    // this.field('projects', { boost: 3 }); // example for an individual toxonomy called projects
+                    this.field('description', { boost: 2 });
+                    this.field('body');
+
+                    data.forEach((doc) => {
+                        this.add(doc);
+
+                        resultDetails.set(doc.ref, {
+                            title: doc.title,
+                            excerpt: doc.excerpt,
+                        });
+                    });
+                });
+
+                $searchInput.trigger('change');
+            }
+        );
+
+        const render = ($targetSearchInput) => {
+            // Dispose the previous result
+            $targetSearchInput.popover('dispose');
+
+            //
+            // Search
+            //
+
+            if (idx === null) {
+                return;
+            }
+
+            const searchQuery = $targetSearchInput.val();
+            if (searchQuery === '') {
+                return;
+            }
+
+            const results = idx
+                .query((q) => {
+                    const tokens = lunr.tokenizer(searchQuery.toLowerCase());
+                    tokens.forEach((token) => {
+                        const queryString = token.toString();
+                        q.term(queryString, {
+                            boost: 100,
+                        });
+                        q.term(queryString, {
+                            wildcard:
+                                lunr.Query.wildcard.LEADING |
+                                lunr.Query.wildcard.TRAILING,
+                            boost: 10,
+                        });
+                        q.term(queryString, {
+                            editDistance: 2,
+                        });
+                    });
+                })
+                .slice(
+                    0,
+                    $targetSearchInput.data('offline-search-max-results')
+                );
+
+            //
+            // Make result html
+            //
+
+            const $html = $('<div>');
+
+            $html.append(
+                $('<div>')
+                    .css({
+                        display: 'flex',
+                        justifyContent: 'space-between',
+                        marginBottom: '1em',
+                    })
+                    .append(
+                        $('<span>')
+                            .text('Search results\xa0') // \xa0 is the Non-breakable space char (&nbsp;)
```

Comment:
This is the only line that is different from the Docsy `assets/js/offline-search.js` file. I'm not sure I'm happy with overriding a whole file for such a small change.

@chalin, @kapunahelewong, do you have any ideas for a better method of making this fix?

Reply:
oof. Yeah, agreed. I don't know though. I'll keep an eye out for a solution. Does the build tree shake duplicate code? If so then it would at least mitigate duplication.

Reply:
For now, add the usual opening line comment stating that it is a Docsy file override.

Reply:
Added the line.

Reply:
> @chalin ... do you have any ideas for a better method of making this fix?

After looking at this more closely, yes 😄 -- see #493.